doc/gh-pages/pages/development/development.md

```diff
@@ -1,10 +1,39 @@
 ---
-title: Develop
+title: Development
 has_children: true
-nav_order: 5
+nav_order: 4
 ---
-# Develop
+# How to contribute to the MNE-CPP Project
+
+## Welcome aboard!
+
+Before you start, there is a few basic concepts that should be understood. We use a variety of different tools to develop and maintain MNE-CPP.
```

Comment:
Should be 'there are a few basic concepts'

Reply:
Thank you.